cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -858,14 +873,18 @@ mod test {
             &src,
         )
         .unwrap();
+        let a_span = Span::new(8, 9);
+        let b_span = Span::new(11, 12);
         assert_eq!(
             grm.prods[grm.get_rule("A").unwrap().pidxs[0]],
             Production {
-                symbols: vec![token("a"), rule("B")],
+                symbols: vec![token("a", Span::new(8, 9)), rule("B", Span::new(11, 12))],
```

Comment:
The span argument to rule is I believe used here, where we are comparing equality with span's generated by the parser.

incidentally this should be `a_span`  and `b_span` instead of `Span::new(...)`

Reply:
If we're only using this in tests, we can make our own helper function `token_eq` which compares spans.

*Alternatively* (and I haven't thought this through!) we could define `Symbol` to be "an instance of a symbol at a given point in a file" at which point `Eq` & `Hash` do make sense.